### PR TITLE
ndx-depth-moseq

### PR DESCRIPTION
**Pull request by pauladkisson**
This extension currently adds a neurodata type for depth moseq's extract step.

### CODE REVIEW

**Comment by rly** [comment]
Hi @pauladkisson, this PR adds two extensions -- did you mean to include ndx-bipolar-referencing?

**Comment by pauladkisson** [comment]
> Hi @pauladkisson, this PR adds two extensions -- did you mean to include ndx-bipolar-referencing?

Not sure why this extension snuck into Master on the catalyst neuro side but I just removed it.

**Comment by rly** [comment]
Sounds good. Please mark as ready for review when ready and I'll take a look.

File: ndx-depth-moseq/ndx-meta.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: ndx-depth-moseq
+version: 0.1.1
```
Comment (rly) [inline ndx-depth-moseq/ndx-meta.yaml:2]:
```suggestion
version: 0.1.2
```

Update to latest version after it is released on PyPI

**Comment by rly** [comment]
Looks good to me, pending a version update. Sorry I missed this after your last comment!

**Comment by pauladkisson** [comment]
> Looks good to me, pending a version update. Sorry I missed this after your last comment!

Done!

**Review by rly — APPROVED** [review]

**Pull request merged**